4/m_whoisgeolite.cpp

```diff
@@ -18,130 +18,90 @@
 
 /// $ModAuthor: reverse <mike.chevronnet@gmail.com>
 /// $ModDesc: Adds city and country information to WHOIS using the MaxMind database.
-/// $ModConfig: <geolite dbpath="conf/geodata/GeoLite2-City.mmdb">
+/// $ModConfig: <geolite dbpath="path/geodata/GeoLite2-City.mmdb">
 /// $ModDepends: core 4
 
-/// $CompilerFlags: find_compiler_flags("libmaxminddb")
-/// $LinkerFlags: find_linker_flags("libmaxminddb")
-
-/// $PackageInfo: require_system("alpine") libmaxminddb-dev pkgconf
-/// $PackageInfo: require_system("arch") pkgconf libmaxminddb
-/// $PackageInfo: require_system("darwin") libmaxminddb pkg-config
-/// $PackageInfo: require_system("debian~") libmaxminddb-dev pkg-config
-/// $PackageInfo: require_system("rhel~") pkg-config libmaxminddb-devel
+/// $LinkerFlags: -lmaxminddb
```

Comment:
Removing these directives will break the CI build and building on distros which don't have a global library path.